src/transformers/integrations/ggml_utils.py

```diff
@@ -0,0 +1,84 @@
+# Constants for ggml imatrix dequantization
+# migrate from https://github.com/ggerganov/ggml/blob/3f5a4bbe59285c0f679b376f6259187d5514ff9c/src/ggml-common.h#L437
+import numpy as np
+
+
+IQ2XXS_GRID = np.array([
+    0x0808080808080808, 0x080808080808082b, 0x0808080808081919, 0x0808080808082b08,
```

Comment:
How can we bypass `make style` for this file? This is copied and translated from source code of `llama.cpp`. I would like to keep the style align with source code.

Reply:
Make sense ! WDYT @ArthurZucker ? Is this something we want ? I also see that there is a `exclude_folders` var in the makefile

Reply:
we usually use `# fmt: skip`